app/src/main/res/layout/fragment_library.xml

```diff
@@ -22,160 +21,136 @@
                 android:layout_height="wrap_content"
                 android:orientation="vertical">
 
-                <com.google.android.material.card.MaterialCardView
-                    style="@style/Widget.Material3.CardView.Elevated"
+                <LinearLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:orientation="horizontal"
+                    android:layout_gravity="center"
+                    android:paddingVertical="6dp">
+
+                    <com.google.android.material.button.MaterialButton
+                        android:id="@+id/watch_history"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:paddingHorizontal="15dp"
+                        android:paddingVertical="15dp"
+                        android:text="@string/watch_history"
+                        android:textAlignment="viewStart"
+                        android:textSize="18sp"
+                        android:textStyle="bold"
+                        app:icon="@drawable/ic_time_outlined"
+                        android:rotation="-5"/>
+
+                    <View
+                        android:layout_width="10dp"
+                        android:layout_height="wrap_content" />
+
+                    <com.google.android.material.button.MaterialButton
+                        android:id="@+id/downloads"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:paddingHorizontal="15dp"
+                        android:paddingVertical="15dp"
+                        android:text="@string/downloads"
+                        android:textSize="18sp"
+                        android:textStyle="bold"
+                        app:icon="@drawable/ic_download"
+                        android:layout_gravity="end"
+                        android:rotation="-5"/>
```

Comment:
Due to their rotation, these are slightly cut off at the top and bottom.

Reply:
Yeah, I haven't figured out how to fix this yet :/